wiki/Mascots/en.md

```diff
@@ -92,7 +92,7 @@ Designed by [pyun](https://osu.ppy.sh/users/981534), Chirou was one of the parti
 
 ![Taikonator](img/Taikonator.png "Taikonator")
 
-Taikonator, also known as Taikonator 3000, was one of the participants in the osu!taiko mascot design contest. He has gained some traction as an inside joke and popularity for unknown reasons. His origins remains a mystery; despite this, he is far more unique than the other osu!taiko mascot design submissions and has made many appearances in various user fanart submissions.
```

Comment:
The first sentence reads so weirdly now. Lemia-Chan was the participant in the contest I think, not Taikonator 3000? Taikonator 3000 would be the "entry" in the contest.

Reply:
i think the wording is supposed to imply some amount of personification of the mascots. for example, the description below this one reads like this:

> Designed by [crystalsuicine](https://osu.ppy.sh/users/9974), Tama was one of the younger participants in the osu!taiko mascot design contest at only age 15 (or is she?), Tama has a huge passion for taiko drumming.

i've applied your suggestion though